src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/EnumExtensions.cs

```diff
@@ -29,5 +29,10 @@ private static EnumLabelValueAttribute GetEnumLabelValueAttribute(this Enum enum
         {
             return enumValue.GetEnumAttribute<EnumLabelValueAttribute>();
         }
+        
+        private static EnumLabelValueAttribute GetEnumLabelAttribute(this Enum enumValue)
```

Comment:
I don't understand this. It seems to be identical to the method above - `GetEnumLabelValueAttribute`

Reply:
done